cmd/internal/find/primary/meta/testdata/before_munging.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
Can you maybe add a README.md in testdata that describes before and after munging? Is there a process you went through to generate them?

Reply:
I was wondering how I should document this. Think a README would make sense. Will add.